common/lib/xmodule/xmodule/course_module.py

```diff
@@ -335,6 +335,15 @@ class CourseFields(object):
         help=_("Enter the unique identifier for your course's video files provided by edX."),
         scope=Scope.settings
     )
+    facebook_url = String(
+        help=_(
+            "Enter the URL for the official course Facebook group. "
+            "If you provide a URL, the edX mobile app includes a button that students can tap to access the group."
+        ),
```

Comment:
I had an old PR where we discussed this exact thing with Lou and Alison.  Let me see if I can find it.
The general outcome of that discussion was that Lou wanted the advanced setting in the course to be specific to Facebook.